scripts/check_fips_compatibility.py

```
@@ -222,76 +277,85 @@ def check_python_file(file_path: Path) -> list[FipsIssue]:
-                            line_number=line_num,
-                            severity="info",
-                            category="package",
-                            message=f"Package may need FIPS verification: {package}",
-                            fix_hint=message,
-                        )
-                    )
+            issues.extend(
+                _check_package_patterns(content, file_path, package, message, "info")
+            )
+
+        return issues
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)